src/fontMetrics.js

```diff
@@ -18,30 +18,28 @@ var cjkRegex = require("./unicodeRegexes").cjkRegex;
 // dimensions, one for each of textstyle, scriptstyle, and scriptscriptstyle,
 // but we only use the textstyle ones, and scale certain dimensions accordingly.
 // See the TeXbook, page 441.
-var sigma1 = 0.025;
-var sigma2 = 0;
-var sigma3 = 0;
-var sigma4 = 0;
-var sigma5 = 0.431;
-var sigma6 = 1;
-var sigma7 = 0;
-var sigma8 = 0.677;
-var sigma9 = 0.394;
-var sigma10 = 0.444;
-var sigma11 = 0.686;
-var sigma12 = 0.345;
-var sigma13 = 0.413;
-var sigma14 = 0.363;
-var sigma15 = 0.289;
-var sigma16 = 0.150;
-var sigma17 = 0.247;
-var sigma18 = 0.386;
-var sigma19 = 0.050;
-var sigma20 = 2.390;
-var sigma21 = 1.01;
-var sigma21Script = 0.81;
-var sigma21ScriptScript = 0.71;
-var sigma22 = 0.250;
+var sigma1 = 0.025;                  // slant
```

Comment:
`tftopl` reported this as 0.25.  It's not actually used anywhere.  I was thinking I should update this.